### DISCUSSION TITLE
Need to use your cloud access

### DISCUSSION THREAD

**Discussion by mdehrlich** [GerryDazoo/Slinger#175] [General]
I thought I had a fixed external IP address.  We had a power failure at 4:30am, and I now find my IP address changed.  I guess I better use finder id to be safe.   Thank you for helping us out with this.

**Comment by mdehrlich** [comment]
As I look into using the cloud access, I have some questions.
1) does this work with unified config.ini
2)I am using two names for each slingbox with different resolutions i.e. MainHD and MainLD.  If usinf finderid, can I have more than one name per box?

**Reply by GerryDazoo** [reply]
1) Yes       Send your request to http://sbfinder.dazoo.net:54321/URLbase/SB_Name/FINDERID
2) Yes

I have a similar setup with two slingbox entries one for high resolution local streaming and one for a low bit rate config I use when viewing on my phone while on the cellular network.

**Comment by mdehrlich** [comment]
Thanks
